packages/slate-plugins/package.json

```diff
@@ -33,6 +34,8 @@
     "is-hotkey": "^0.1.6",
     "lodash": "^4.17.15",
     "prismjs": "^1.20.0",
+    "react-dnd": "^11.1.3",
+    "react-dnd-html5-backend": "^11.1.3",
```

Comment:
This repository is archived. in the repo readme says all the code has being merged to `react-dnd`
https://github.com/react-dnd/react-dnd-html5-backend

Reply:
OK now I got it, the code is now in the monorepo but the package name is the same. so no need of a change on this one!

Reply:
Indeed, but we still need to install the package: https://react-dnd.github.io/react-dnd/about